### ISSUE TITLE
Written communication is remote work super power

### ISSUE THREAD

**Issue by jprudent** [jprudent/jprudent.github.com#29]
---
- url: https://snir.dev/blog/remote-async-communication/
---

chat problems
===
- talking is easier than chatting (you loose your time formulating and typing)
- chatting is too easy to do (you get spam)
- everybody expects immediate response
- it promotes secrecy (private conversation)

Videoconference problems
===
- 
- talky walky mode (1 speaker at a time)
- no side-talks possible
- feels like a lecture
- are tiring (sound, ...)

Synchronous communication problems
===
- expect immediate response
- bad work culture
  - cost of interupting someone
  - if you can ask, why write documentation ?
  - no planning : when problems arise, just ask
- a culture of interuption that feeds itself
- no worktime flexibility because everyone is dependant on everyone

Async writing can solve this
===
- async writing lives longer, better targetting, easy to discover
- it gives context (eg. the mysql user database)
- not contextual to an event (eg. the db))
![image](https://user-images.githubusercontent.com/270929/85232652-d8e35c00-b400-11ea-8f06-f0907a173874.png)
- need organization
  - full text is bad (no incentive to organization)
  - tags, templates, ...

TL;DR
===
- chats and videos are not office conversations and they are tiring
- async enables
  - keep on workflow, no context switch, independant of others
  - a documentation that stays, is refined
